src/earn/EarnCard.tsx

```diff
@@ -14,9 +15,16 @@ interface Props {
 }
 
 export function EarnCardDiscover({ depositTokenId, poolTokenId }: Props) {
-  const showStablecoinEarn = getFeatureGate(StatsigFeatureGates.SHOW_STABLECOIN_EARN)
+  const showMultiplePools = getFeatureGate(StatsigFeatureGates.SHOW_MULTIPLE_EARN_POOLS)
   const poolToken = useTokenInfo(poolTokenId)
 
+  if (showMultiplePools) {
+    // TODO(ACT-1257): show active pools
+    return <EarnEntrypoint />
+  }
+
+  const showStablecoinEarn = getFeatureGate(StatsigFeatureGates.SHOW_STABLECOIN_EARN)
```

Comment:
Is `showStablecoinEarn` only going to be used to refer to the MVP now? I'm just wondering if down the road there could be some case where `showStablecoinEarn` is false but `showMultiplePools` is true and if that would have any issues

Reply:
yes that's an MVP specific gate. If we wanted to combine gates, we can always do it from statsig, instead of enforcing in code